src/crypto/CHIPCryptoPALOpenSSL.cpp

```diff
@@ -1940,5 +1941,80 @@ CHIP_ERROR ExtractVIDPIDFromX509Cert(const ByteSpan & certificate, AttestationCe
     return err;
 }
 
+/**
+ *  @brief Openssl P256 key builder for Operatinal and Ephemeral keys.
+ **/
+class OpensslDefaultP256KeypairBuilder : public P256KeypairBuilder
+{
+public:
+    virtual P256Keypair * BuildP256KeyPairForOperationalKey(FabricIndex fabricIdx) override;
+
+    virtual CHIP_ERROR ConfirmP256KeyPairForOperationalKey(FabricIndex fabricIdx, P256Keypair * p256key) override;
+
+    virtual P256Keypair * GetP256KeyPairForOperationalKey(FabricIndex fabricIdx) override;
+
+    virtual P256Keypair * BuildP256KeyPairForEphemeralUsage() override;
+
+    virtual void FreeP256KeyPair(P256Keypair * p256key) override;
+};
+
+P256Keypair * OpensslDefaultP256KeypairBuilder::BuildP256KeyPairForOperationalKey(FabricIndex fabricIdx)
+{
+    (void) fabricIdx;
+    P256Keypair * keypair = nullptr;
+    keypair               = Platform::New<P256Keypair>();
+    keypair->Initialize();
+    return keypair;
+}
+
+CHIP_ERROR OpensslDefaultP256KeypairBuilder::ConfirmP256KeyPairForOperationalKey(FabricIndex fabricIdx, P256Keypair * p256key)
+{
+    // TBD
+    (void) fabricIdx;
+    (void) p256key;
+    return CHIP_NO_ERROR;
+}
+
+P256Keypair * OpensslDefaultP256KeypairBuilder::GetP256KeyPairForOperationalKey(FabricIndex fabricIdx)
+{
+    // TBD
+    (void) fabricIdx;
+    return NULL;
```

Comment:
use `nullptr`.

Why are we adding an empty implementation of things for openssl? please add a description of this in the summary - the title of the bug and summary says "remove use by value" however the description does not seem to explain the changes done. Ideally I would like to read the PR summary and understand exactly what to expect in the PR body.